### DISCUSSION TITLE
Problema com Bluetooth

### DISCUSSION THREAD

**Discussion by emerson-molusco** [blueman-project/blueman#2460] [Q&A]
blueman: 
BlueZ:
Distribution: Endeavour 
![Captura de tela de 2024-07-01 09-36-40](https://github.com/blueman-project/blueman/assets/174339492/7941621e-06d7-4d18-baa5-2b74aca7b325)

Desktop environment: Cinnamon

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

**Comment by cschramm** [comment]
`systemctl status bluetooth.service`

**Comment by emerson-molusco** [comment]
![bluetooth_erro](https://github.com/user-attachments/assets/fe8b6dc1-95a7-4280-93a8-0b79949ccec7)


> `systemctl status bluetooth.service`

deu isso.

**Comment by cschramm** [comment]
Well, from what I see it does not seem to be running.

**Comment by emerson-molusco** [comment]
what i have to do??

In fact, i dont know if the machine has the component necessary to works
???

**Reply by cschramm** [reply]
I cannot even see the logs in your screenshot and fail to make sense of "service wa[...]", but it states that it has no running processes ("inactive (dead)"), so it makes perfect sense that blueman cannot reach it. At least with recent systemd, the unit typically logs at least the following messages per cycle:

```
Starting Bluetooth service...
Started Bluetooth service.
Stopping Bluetooth service...
bluetooth.service: Deactivated successfully.
Stopped Bluetooth service.
```

Plus, quite some messages from bluetoothd itself, at least a version.

A single message per invocation(?) seems to indicate an issue when starting the service or even the reason why it's not started.

Your distribution's support / community might fit better for troubleshooting bluetooth.service.
